### PR TITLE
Code Review 

### PR DESCRIPTION
**Pull request by ggotora**

### CODE REVIEW

**Comment by kitpao** [comment]
# 👋 Hi @ggotora  and @IjayAbby ,

:pencil: Friendy reminder: Remember it's important to have a professional repo so you can increase your job opportunities.
Please take a step back and read the lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Sadly for now, your review has been marked as **invalid** because of the following reason:
- [ ] You are missing a short summary of your changes in the pull request.

Once you correct this, please submit a new code review and we will be glad to check it :)
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao
##### 👉 You can find me in your completition form as Kitzia Paola Vidal Marroquin

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Comment by IjayAbby** [comment]
In this development branch, we created a form using HTML & CSS which was the second Microverse Project in the HTML & CSS course.
The Project required us to create a replica of the SignUp form in Mint.com or alternatively Mailchimp.com

**Comment by IjayAbby** [comment]
Fixed the README.MD file and CSS file errors.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                <p class="small">Standard call, messaging or data rates may apply</p>
+                <label for="password">Password</label>
+                <input type="password" id="password" autocomplete="current-password">
+                <span class="pwd-key">&#128275;</span>
+                <button class="btn" type="submit"><span>&#128274;</span> Create Account</button>
+                <p class="consent zero-mgn">By clicking Create account, you agree to our
+                    Terms and have read and acknowledge our Privacy Statement.</p>
+            </div>
+            <p class="form-footer center">
+                Invisible reCAPTCHA by Google Privacy Policy and Terms of Use.
```
Comment (krys2fa) [inline index.html:45]: - [ ] [OPTIONAL] The words `Privacy Policy` and `Terms of Use` should be web links as seen on the [original site](https://accounts.intuit.com/signup.html)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <label for="email">Email Address</label>
+                <input type="email" autocomplete="email">
+                <label for="tel">Phone (recommended)</label>
+                <input type="tel" id="tel" autocomplete="tel">
+                <p class="small">Standard call, messaging or data rates may apply</p>
+                <label for="password">Password</label>
+                <input type="password" id="password" autocomplete="current-password">
+                <span class="pwd-key">&#128275;</span>
+                <button class="btn" type="submit"><span>&#128274;</span> Create Account</button>
+                <p class="consent zero-mgn">By clicking Create account, you agree to our
```
Comment (krys2fa) [inline index.html:41]: - [ ] [OPTIONAL] The words `Terms` and `Privacy Statement` should be web links as seen on the [original site](https://accounts.intuit.com/signup.html)

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Awesome work @ggotora and @IjayAbby 👏 
Your project is a really great clone of the original site with good use of semantic HTML as well as CSS classes. 💪 
Good job on implementing the changes suggested by the previous TSE as well. 👍 
Just a few more adjustments to complete the project.
### Changes Required ♻️ 

- [ ] Kindly fix all linter errors. For example, you are using an indentation of more than 2 spaces in your `style.css` file.

- [ ] Also, check out comments below to add links to a few words at the bottom of the form.

Please request for another review after implementing the changes.

Happy Coding 🖥️ 😃 
Chris

###### You can contact me on slack (@ Chris Amanor) for any clarification.

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :green_circle: 

#### Great work Team :clap:

The design looks really great and you implemented the requested changes by my colleagues above. Go ahead and merge :100: 

Good Luck and Happy Coding :+1: :

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by IjayAbby** [comment]
The project has been approved and the configuration files are working perfectly.
The reviewers gave helpful information on the areas to be worked on.

**Pull request merged**